Testnet/NonFungibleToken/NonFungibleToken/NonFungibleToken.cs

```diff
@@ -36,148 +34,175 @@ public struct ApprovalForAllLog
         public bool Approved;
     }
 
+    public struct OwnershipTransferedLog
+    {
+        [Index]
+        public Address PreviousOwner;
+
+        [Index]
+        public Address NewOwner;
+    }
+
     /// <summary>
-    /// Get a value indicacting if the interface is supported.
+    /// Function to check which interfaces are supported by this contract.
     /// </summary>
-    /// <param name="interfaceId">The id of the interface to support.</param>
-    /// <returns>A value indicating if the interface is supported.</returns>
-    private bool GetSupportedInterfaces(uint interfaceId)
+    /// <param name="interfaceID">Id of the interface.</param>
+    /// <returns>True if <see cref="interfaceID"/> is supported, false otherwise.</returns>
+    public bool SupportsInterface(uint interfaceID)
     {
-        return this.PersistentState.GetBool($"SupportedInterface:{interfaceId}");
+        return State.GetBool($"SupportedInterface:{interfaceID}");
     }
 
     /// <summary>
     /// Sets the supported interface value.
     /// </summary>
     /// <param name="interfaceId">The interface id.</param>
     /// <param name="value">A value indicating if the interface id is supported.</param>
-    private void SetSupportedInterfaces(uint interfaceId, bool value)
-    {
-        this.PersistentState.SetBool($"SupportedInterface:{interfaceId}", value);
-    }
+    private void SetSupportedInterfaces(uint interfaceId, bool value) => State.SetBool($"SupportedInterface:{interfaceId}", value);
 
     /// <summary>
     /// Gets the key to the persistent state for the owner by NFT ID.
     /// </summary>
     /// <param name="id">The NFT ID.</param>
     /// <returns>The persistent storage key to get or set the NFT owner.</returns>
-    private string GetIdToOwnerKey(ulong id)
-    {
-        return $"IdToOwner:{id}";
-    }
+    private string GetIdToOwnerKey(ulong id) => $"IdToOwner:{id}";
 
     ///<summary>
     /// Gets the address of the owner of the NFT ID. 
     ///</summary>
     /// <param name="id">The ID of the NFT</param>
     ///<returns>The owner address.</returns>
-    private Address GetIdToOwner(ulong id)
-    {
-        return this.PersistentState.GetAddress(GetIdToOwnerKey(id));
-    }
+    private Address GetIdToOwner(ulong id) => State.GetAddress(GetIdToOwnerKey(id));
```

Comment:
make public

Reply:
These are tiny wrappers for avoid magic strings and easy access to state variables. Any required information is available as methods already. Secondly we only want to expose ERC721 related interfaces only. 

Please see OwnerOf method.